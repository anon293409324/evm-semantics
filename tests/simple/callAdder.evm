account:
-   id: 1
-   balance: 40
-   program: PUSH 30 ; PUSH 20 ; PUSH 0 ; MSTORE ; PUSH 1 ; MSTORE ;
             PUSH 1 ; PUSH 3 ; PUSH 2 ; PUSH 0 ; PUSH 400 ; PUSH 2 ; CALL
-   storage: 30 , 20 , 25

account:
-   id: 2
-   balance: 70
-   program: PUSH 0 ; MLOAD ; PUSH 1 ; MLOAD ; ADD ; PUSH 0 ; MSTORE ; PUSH 1 ; PUSH 0 ; RETURN
-   storage: 22 , 50 , 70

.Accounts


transaction:
-   to: 1
-   from: .AcctID
-   data:
-   value: 0
-   gasPrice: 1
-   gasLimit: 1000

.Transactions
