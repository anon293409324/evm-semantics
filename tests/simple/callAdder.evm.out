<T> <k> . </k> <id> 1 </id> <pc> 13 </pc> <gas> 264 </gas> <wordStack> .WordStack </wordStack> <localMem> 0 |-> 20 1 |-> 30 3 |-> 50 </localMem> <gasPrice> 1 </gasPrice> <callStack> .CallStack </callStack> <accounts> <account> <acctID> 2 </acctID> <program> 0 |-> ( PUSH 0 ) 1 |-> MLOAD 2 |-> ( PUSH 1 ) 3 |-> MLOAD 4 |-> ADD 5 |-> ( PUSH 0 ) 6 |-> MSTORE 7 |-> ( PUSH 1 ) 8 |-> ( PUSH 0 ) 9 |-> RETURN </program> <storage> 0 |-> 22 1 |-> 50 2 |-> 70 </storage> <balance> 70 </balance> </account> <account> <acctID> 1 </acctID> <program> 0 |-> ( PUSH 30 ) 1 |-> ( PUSH 20 ) 2 |-> ( PUSH 0 ) 3 |-> MSTORE 4 |-> ( PUSH 1 ) 5 |-> MSTORE 6 |-> ( PUSH 1 ) 7 |-> ( PUSH 3 ) 8 |-> ( PUSH 2 ) 9 |-> ( PUSH 0 ) 10 |-> ( PUSH 400 ) 11 |-> ( PUSH 2 ) 12 |-> CALL </program> <storage> 0 |-> 30 1 |-> 20 2 |-> 25 </storage> <balance> 40 </balance> </account> </accounts> </T>
